libqtile/widget/battery.py

```diff
@@ -233,9 +233,11 @@ def _load_file(self, name) -> Optional[Tuple[str, str]]:
         try:
             with open(path, 'r') as f:
                 return f.read().strip(), value_type
-        except FileNotFoundError:
-            logger.debug("Failed to get %s" % name)
-        return None
+        except OSError as e:
+            logger.debug("Failed to read '{}': {}".format(path, e))
+            if isinstance(e, FileNotFoundError):
+                return None
+            return "-1", "N/A"
```

Comment:
You could do this as:
```
except FileNotFoundError:
    logger.debug("Failed to get %s" % name)
    return None
except OSError:
    return "-1", "N/A"
```

Reply:
Yeah, I just wanted to log in any case without copy pasting the said line.